_workinggroups/training.md

```diff
@@ -50,7 +50,7 @@ The [HSF-Training GitHub Organization](https://github.com/hsf-training) has [Ana
 {% for post in schools %}
   {% capture date %}{{post.end_date | date: '%s' | plus: 0 }}{% endcapture %}
   {% if date > now %}
-  1. [**{{post.date | date: "%-d %b"}} - {{post.end_date | date: "%-d %b %Y"}}** - {{post.title}} ]({{post.source}})
+  1. [**{{post.date | date: "%-d %b"}} - {{post.end_date | date: "%-d %b %Y"}}** - {{post.title}} - **Deadline:** {{post.deadline}} ]({{post.source}})
```

Comment:
Please add an if statement that it only shows the ``Deadline:``, if a deadline has been configured

Reply:
Regarding the `if statement`
I have implemented the change but it produces an error that I think means that the object `post.deadline` is  an "nil" or false
```
{% for post in schools %}
  {% capture date %}{{post.end_date | date: '%s' | plus: 0 }}{% endcapture %}
  {% if date > now %}
  {%if {{post.deadline}} %}
  1. [**{{post.date | date: "%-d %b"}} - {{post.end_date | date: "%-d %b %Y"}}** - {{post.title}} - **Deadline:** {{post.deadline}} ]({{post.source}})
  {% else %}
  1. [**{{post.date | date: "%-d %b"}} - {{post.end_date | date: "%-d %b %Y"}}** - {{post.title}} ]({{post.source}})
  {% endif %}
  {% endif %}
{% endfor %}
```

![image](https://user-images.githubusercontent.com/17858942/74610318-8f88a600-50c8-11ea-96a4-494118382168.png)

Reply:
hi, this seems to be a jekyll bug as seen here: https://github.com/jekyll/jekyll/issues/7380

Adding ``excerpt: none`` to the frontmatter solves this for me.

Reply:
Thanks so much. Now I believe we need to either remove the `deadine: ''` attribute altogether or just leave it as `deadline: ` in order to get the correct "Truthy" or "Falsy" that jekyll is looking for.

Reply:
I have tried both of these solutions and also commenting on the line where `deadline: ''` is found and all of them work great. I will proceed to just comment them out.

Reply:
Hmm, I think it's not elegant to comment out the deadline field. The if statement should be `` {% if post.deadline != blank %}`` then it works.

Reply:
Done!